examples/from_log.rs

```diff
@@ -17,7 +17,7 @@ fn main() -> Result<()> {
     let input = File::open(path)?;
     let buffered = BufReader::new(input);
 
-    let mut map = LcsMap::new();
+    let mut map = LcsMap::new(None);
```

Comment:
Is there any specific reason to use `&[&str]` as argument in your suggestion?

Reply:
Hum I don't think so... I wanted to ask for `from_delimiters()` but for the argument I guess a better one could be used.

Do you have a suggestion? That's actually something I'm interested to learn more; how to pass something as generic as possible, similar to `AsRef<str>` but for a slice?

Reply:
Well, I am using simply a vector, so that type becomes serialisable. I literally started to code in rust now. :) have not much experience.

On `from_*`: in the guideline it says that `from_*` should be used when converting from something to this type, which does not happening in this case, so I would suggest either:

```rust
let map = LcsMap::with_delimiters(vec![' ', ',']);
```

or a type builder

```rust
let map = LcsMap::new().delimiters(vec![' ', ',']);
```

What do you think?

Reply:
Those are two good ideas!

Both could be added, but I think one should be enough.

Since there is a single option (the delimiters), I think I would prefer the `with_delimiters()` variant.